src/commcare_cloud/environment/schemas/postgresql.py

```diff
@@ -53,15 +53,16 @@ def validate_shards(shard_ranges_by_partition_name):
     assert _is_power_of_2(num_shards), \
         'Total number of shards must be a power of 2: {}'.format(num_shards)
 
+DEFAULT_POSTGRESQL_USER = "{{ secrets.POSTGRES_USERS.commcare.username }}"
```

Comment:
How come you moved this here?

Reply:
Just to make it a constant

Reply:
Yeah the idea was to make it not part of the schema anymore so that people can't declare it in `postgresql.yml`. Pulled out here, it's just a constant that I can use later in the code.